src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss

```diff
@@ -0,0 +1,63 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+.card-wrapper {
+    max-width: 320px;
+    min-width: 220px;
+}
+
+$dark-card: igx-card-theme(
+    $background:#fd6bd93a,
+    $header-text-color:#f2f607,
+    $subtitle-text-color: #fa2509,
+    $content-text-color:#0a41f7
+);
+
+/*ENHANCED APPROACHES*/
+
+// $yellow-color: #e7eb0b;
+// $black-color: #f00b1e;
```

Comment:
This color is not black. maybe we should rename the variable.

Reply:
Done